README.md

```diff
@@ -1 +1,111 @@
-# My-Portfolio-desktop-version
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
```

Comment:
- [OPTIONAL] The particular heading is missing in your README file. Kindly add it and its appropriate information under it. I made it optional because it was not mentioned by the previous reviewer. But it is highly recommended to have it in your README file.